rosbag2_py/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rosbag2_py</name>
+  <version>0.2.4</version>
+  <description>Python API for rosbag2</description>
+  <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
+  <maintainer email="ros-tooling@googlegroups.com">ROS Tooling Working Group</maintainer>
+  <license>Apache License 2.0</license>
+
```

Comment:
@mabelzhang You can add yourself as an author.

Reply:
Hmm I don't know if I want to officially sign up for that...

Reply:
It doesn't imply maintainership, rather giving you credit (use the `<author>` tag). Up to you though.

Reply:
Turns out the author tag isn't allowed by the linter. So I'll leave it off. Thanks for mentioning though!

Reply:
> Turns out the author tag isn't allowed by the linter.

That's surprising. We use `<author>` tags in several other core ROS 2 packages.

Reply:
> Turns out the author tag isn't allowed by the linter.

You need to follow the order of the tags defined by the schema to successfully validate the XML file.